pyaedt/icepak.py

```diff
@@ -3930,21 +3932,18 @@ def assign_solid_block(
             ``"Joule Heating"``. For a temperature-dependent or transient
             assignment, a dictionary can be used. The dictionary should contain three keys:
             ``"Type"``, ``"Function"``, and ``"Values"``.
-
```

Comment:
Removing the blank line means that the bulleted list will not render properly in the documentation. You MUST have a blank line before the first item in the list.